assets/queries/terraform/gcp/sql_db_instance_with_unrecommended_error_logging_threshold/metadata.json

```
@@ -0,0 +1,14 @@
+{
+  "id": "13de4e49-d407-4277-ba5a-d7f59283902f",
+  "queryName": "Beta - SQL DB Instance With Unrecommended Error Logging Threshold",
+  "severity": "LOW",
+  "category": "Observability",
+  "descriptionText": "All 'google_sql_database_instance' resources based on POSTGRES should have the 'log_min_error_statement' flag set to 'ERROR' or a higher severity to prevent excessively logging",
```

I would suggest changing the word excessively to excessive.
```suggestion
  "descriptionText": "All 'google_sql_database_instance' resources based on POSTGRES should have the 'log_min_error_statement' flag set to 'ERROR' or a higher severity to prevent excessive logging",
```